packages/aws-cdk/lib/api/hotswap-deployments.ts

```diff
@@ -40,7 +39,7 @@ export async function tryHotswapDeployment(
     account: resolvedEnv.account,
     region: resolvedEnv.region,
     partition: (await sdk.currentAccount()).partition,
-    urlSuffix: sdk.getEndpointSuffix,
+    urlSuffix: sdk.getEndpointSuffix(resolvedEnv.region),
```

Comment:
Nope! Please, leave this as-is, and pass a lambda function there, like Rico suggested.

This is so that we avoid as many service calls as we can during hotswapping.

Reply:
ahhh sorry, fixed!